packages/graph-explorer/src/modules/KeywordSearch/KeywordSearch.tsx

```diff
@@ -342,6 +343,18 @@ const KeywordSearch = ({
                     )}
                     {currentTotal != null && " records "}
                     for matching results
+                    <div
+                      style={{
+                        display: "flex",
+                        flexDirection: "column",
+                        alignItems: "center",
+                        justifyContent: "center",
+                        marginTop: "24px",
+                        gap: "24px",
+                      }}
+                    >
+                      <Button onPress={() => cancelAll()}>Cancel</Button>
```

Comment:
nit: Keep it consistent with other buttons, i.e.
<Button onPress={onCancel}>Cancel</Button>

Reply:
I took a look at the buttons in this view and there is a mix of both inline closures and named closures throughout the file. There is no consistency.

I don’t think the indirection is necessary for this simple case.